### ISSUE TITLE
Add Drag Roboter

### ISSUE THREAD

**Issue by ghost** [viktorradnai/flightgear-ask21#78]
Add a "roboter" aircraft towing you (auto controlled, does a normal circuit).
see https://www.youtube.com/watch?v=gRQzOMJMu7s for how it's done in Condor Soaring
(proposal by Thomas Polzer)

**Comment by viktorradnai** [comment]
Good idea. The DG-101G has one but I couldn't get it working as it kept exceeding max tow force when it started rolling. I'll look into copying the logic.

**Comment by ghost** [comment]
I'll try to do this as well, just for fun ;)

**Comment by viktorradnai** [comment]
@D-ECHO I have taken a shot at the code taken from the DG101 but at some point it's looking for the glider's position in the global coordinate system (not lon/lat) and I haven't found a way to obtain that yet. Some googling would probably yield a solution though.

**Comment by ghost** [comment]
@viktorradnai I have added a test version of a drag robot, adapted from the dg-101g in the new branch dragrobot Please test. At the moment, we need to change the hitch.nas file in fgdata for this to work, so that it supports "dragger" apart from "ai" and "multiplayer"

**Comment by ghost** [comment]
Ooops didn't mean to close ;)

**Comment by ghost** [comment]
Another problem in the current version is that the first-spawned drag roboter can not be removed, all others are disappearing fine and I can't really find the cause
